packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -204,27 +205,33 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
         error={isLargeMobile && error}
         label={isLargeMobile && label}
         disabled={disabled}
-        onFocus={handleFocus}
+        onFocus={!readOnly ? handleFocus : undefined}
         onBlur={handleBlur}
-        onChange={handleInputChange}
+        onChange={!readOnly ? handleInputChange : undefined}
```

Comment:
Do we need this? A readOnly input should not have `onChange` 🤔 could you try it?

Reply:
I will check it and remove if doable.